vaadin-grid-flow-parent/vaadin-grid-testbench/src/main/java/com/vaadin/flow/component/grid/testbench/GridElement.java

```diff
@@ -115,7 +175,7 @@ public GridTHTDElement getCell(int rowIndex, int colIndex) {
     /**
      * Gets the grid cell for the given row and column.
      * <p>
-     * Automatically scrolls the given row into view
+     * Automatically scrolls the given row and column into view
```

Comment:
This JavaDoc update should also be applied to the [JavaDoc of the other overload](https://github.com/vaadin/flow-components/pull/8047/files#diff-69fc125b975a99479abf8b7279f6ca110ba817f00e5246adae46fdc4dbc74014L102).